### PR TITLE
Users/roniga/sessionvariables

### PR DESCRIPTION
**Pull request by roniga**
Adding Session variables support for PowershellOnTargetMachine Task. Fixing loc.json of cmdline and shellscript task

### CODE REVIEW

File: Tasks/PowerShellOnTargetMachines/task.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
             "label": "Machine Group",
             "defaultValue": "",
             "required": true,
-            "helpMarkDown": "Name of the Machine Group for deploying application.​"
+            "helpMarkDown": "Name of the Machine Group for deploying application.?"
```
Comment (kmkumaran) [inline Tasks/PowerShellOnTargetMachines/task.json]: Question mark in the end

File: Tasks/PowerShellOnTargetMachines/task.json [hunk 2]
```diff
@@ -72,6 +72,15 @@
...
             "helpMarkDown": "Setting it to true will run powershell scripts in parallel on the target machines."
+        },
+	    {
+            "name": "SessionVariables",
+            "type": "multiLine",
+            "label": "Session Variables",
+            "defaultValue": "",
+            "required": false,
+	        "groupName":"advanced",
+            "helpMarkDown": "Set session variables which would be available for both the scripts. Example. $variable = value, $var1 = \"value, 123\""
```
Comment (kmkumaran) [inline Tasks/PowerShellOnTargetMachines/task.json]: Have you got the string reviewed with PM?

File: Tasks/CmdLine/task.loc.json [hunk 3]
```diff
@@ -16,7 +16,7 @@
   "version": {
     "Major": 1,
     "Minor": 0,
-    "Patch": 8
+    "Patch": 10
```
Comment (kmkumaran) [inline Tasks/CmdLine/task.loc.json]: I believe this file is not part of your changes and for merge.

File: Tasks/ShellScript/task.loc.json [hunk 4]
```diff
@@ -15,7 +15,7 @@
   "version": {
     "Major": 1,
     "Minor": 0,
-    "Patch": 6
+    "Patch": 8
```
Comment (kmkumaran) [inline Tasks/ShellScript/task.loc.json]: I believe this file is not part of your changes and for merge.

File: Tasks/CmdLine/task.loc.json (hunk 3 above)
Comment (roniga) [inline Tasks/CmdLine/task.loc.json]: Yes, this change was done by @bryanmacfarlane and corresponding changes in the loc.json were missing. I feel this should be merged.

File: Tasks/ShellScript/task.loc.json (hunk 4 above)
Comment (roniga) [inline Tasks/ShellScript/task.loc.json]: Commented above.

File: Tasks/CmdLine/task.loc.json (hunk 3 above)
Comment (bryanmacfarlane) [inline Tasks/CmdLine/task.loc.json]: agreed

**Pull request merged**